op-node/rollup/derive/ithmus_upgrade_transactions.go

```diff
@@ -0,0 +1,198 @@
146102f157600080fd5b80634397dfef1161013a57806354fd4d501161011457806354fd4d5014610259578063550fcdc91461029b5780635cf24969146102d457600080fd5b80634397dfef14610204578063440a5e201461022c5780634d5d9a2a1461023457600080fd5b806309bd5a601161016b57806309bd5a60146101a457806316d3bc7f146101c057806321326849146101f557600080fd5b8063015d8eb914610187578063098999be1461019c575b600080fd5b61019a6101953660046105b8565b6103e0565b005b61019a61051f565b6101ad60025481565b6040519081526020015b60405180910390f35b6008546101dc90640100000000900467ffffffffffffffff1681565b60405167ffffffffffffffff90911681526020016101b7565b604051600081526020016101b7565b6040805173eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee815260126020820152016101b7565b61019a610529565b6008546102449063ffffffff1681565b60405163ffffffff90911681526020016101b7565b60408051808201909152600c81527f312e352e312d626574612e36000000000000000000000000000000000000000060208201525b6040516101b7919061062a565b60408051808201909152600381527f4554480000000000000000000000000000000000000000000000000000000000602082015261028e565b6101ad60015481565b6003546101dc9067ffffffffffffffff1681565b6003546102449068010000000000000000900463ffffffff1681565b6000546101dc9067ffffffffffffffff1681565b6101ad60055481565b6101ad60065481565b6000546101dc9068010000000000000000900467ffffffffffffffff1681565b600354610244906c01000000000000000000000000900463ffffffff1681565b60408051808201909152600581527f4574686572000000000000000000000000000000000000000000000000000000602082015261028e565b60405173deaddeaddeaddeaddeaddeaddeaddeaddead000181526020016101b7565b6101ad60045481565b6101ad60075481565b3373deaddeaddeaddeaddeaddeaddeaddeaddead000114610487576040517f08c379a000000000000000000000000000000000000000000000000000000000815260206004820152603b60248201527f4c31426c6f636b3a206f6e6c7920746865206465706f7369746f72206163636f60448201527f756e742063616e20736574204c3120626c6f636b2076616c7565730000000000606482015260840160405180910390fd5b6000805467ffffffffffffffff98891668010000000000000000027fffffffffffffffffffffffffffffffff00000000000000000000000000000000909116998916999099179890981790975560019490945560029290925560038054919094167fffffffffffffffffffffffffffffffffffffffffffffffff00000000000000009190911617909255600491909155600555600655565b610527610531565b565b610527610544565b610539610544565b60a43560a01c600855565b73deaddeaddeaddeaddeaddeaddeaddeaddead000133811461056e57633cc50b456000526004601cfd5b60043560801c60035560143560801c60005560243560015560443560075560643560025560843560045550565b803567ffffffffffffffff811681146105b357600080fd5b919050565b600080600080600080600080610100898b0312156105d557600080fd5b6105de8961059b565b97506105ec60208a0161059b565b9650604089013595506060890135945061060860808a0161059b565b979a969950949793969560a0850135955060c08501359460e001359350915050565b600060208083528351808285015260005b818110156106575785810183015185820160400152820161063b565b81811115610669576000604083870101525b50601f017fffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffe01692909201604001939250505056fea164736f6c634300080f000a")
```

Comment:
What commit did you pull this bytecode from? Given https://github.com/ethereum-optimism/specs/pull/382/files#r1936336796 its impossible to recreate without brute forcing